### PR TITLE
Merge/1.5.10

### PR DESCRIPTION
**Pull request by StabbyCutyou**
ping @tannerburson @mattcarbone @jjrussell for people who have worked on this with changes in the merge

ping @Tapjoy/eng-group-services since we own this and should make sure people are comfortable with whats being merged.

### CODE REVIEW

File: lib/chore/queues/sqs/consumer.rb [hunk 1]
```diff
@@ -34,10 +34,10 @@ def consume(&handler)
           while running?
             begin
               messages = handle_messages(&handler)
-              sleep 1 if messages.empty?
+              sleep (Chore.config.consumer_sleep_interval || 1) if messages.empty?
             rescue AWS::SQS::Errors::NonExistentQueue => e
-              Chore.logger.error "You specified a queue that does not exist. You must create the queue before starting Chore. Shutting down..."
-              raise Chore::TerribleMistake  
+              Chore.logger.error "You specified a queue '#{queue_name}' that does not exist. You must create the queue before starting Chore. Shutting down..."
```
Comment (jjrussell) [inline lib/chore/queues/sqs/consumer.rb]: This line right here is the best change I've ever seen. Truly sublime.

**Comment by jjrussell** [comment]
my part lgtm. 

other changes also lgtm

File: lib/chore/queues/sqs/consumer.rb (hunk 1 above)
Comment (StabbyCutyou) [inline lib/chore/queues/sqs/consumer.rb]: A+ work on that one

**Comment by tannerburson** [comment]
Yep. Looks good to me as well.

**Comment by StabbyCutyou** [comment]
I'm gonna give the slackers who aren't checking their commits over a
holiday weekend the chance to chime in prior to merging.

**Comment by mattcarbone** [comment]
lgtm :+1:

**Comment by tannerburson** [comment]
Eh?

**Comment by jjrussell** [comment]
Why are you still following our pull requests?

**Comment by jjrussell** [comment]
Don't you have DOTA 2 to play or something?

**Comment by StabbyCutyou** [comment]
This is public, he's able to see it

I'm gonna merge this guy in this afternoon and do a tag + release to ruby gems afterward

**Comment by StabbyCutyou** [comment]
Actually - by now everyone has seen it who contributed.

Lets get this puppy merged

**Comment by StabbyCutyou** [comment]
Done and done.

Happy Choring, all

**Pull request merged**